### DISCUSSION TITLE
missing alignment xml tag for Netflix dfxp

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#4974] [Q&A]
Hello,

When I am writing this, Netflix has already started adopting alignments other than center bottom.
https://www.w3.org/TR/ttml1/#style-attribute-textAlign
tts namespace has an attribute: textalign.

Now I was trying to work on a project where I needed to use this.
However SE is capable of using an# tags to align, but it is not able to export the same to dfxp.

Hence, loading of the file with an# tags back to SE fails to save the positions.

Am I right?

**Comment by niksedk** [comment]
The format `Netflix Timed Text` supports `Bottom center` and `Top center` in SE (so the tag `{\an8}` is supported).

I saw that the TV show `Lucifer` used/uses positioning - it's the only show I've seen with sub positioning (at least with English/Danish subtitles). 
Probably seen a line or two top centered during start credits.

**Reply by PreetM07** [reply]
It is a new adaptation I think they are going towards.
It is not yet mentioned in any of their styleguide, but there are a few shows that has adapted this SG into. Best example is F.R.I.E.N.D.S
Also some new releases like the boy next door, backstage,... etc

I am just curious if we can export it in ttml xml as there is a provision in attributes

**Reply by niksedk** [reply]
Could you attach one (partially is fine) of these raw sub files, so we can see how the positions are saved?

**Reply by PreetM07** [reply]
![image](https://user-images.githubusercontent.com/63149170/114351953-e4b72f00-9b88-11eb-80dd-1030c388b701.png)

**Comment by PreetM07** [comment]
This is from SE
Even though positions are there. They don't get exported.

**Reply by PreetM07** [reply]
Oh. You can't paste the image also?
![image](https://user-images.githubusercontent.com/63149170/114365058-b12fd100-9b97-11eb-99a4-dc58cd797abc.png)

**Reply by PreetM07** [reply]
This is from EZTitles. 
Can you please verify, if .cap has provisions of alignments?
.scc I can see no such things from the link you provided. also please verify?

**Reply by PreetM07** [reply]
I have verified with .cap and .scc and alignments
They have the provisions.
SE can read scc positions.
but not .cap positions. (tested with files exported from EZ)

**Reply by PreetM07** [reply]
> Oh. You can't paste the image also?
> ![image](https://user-images.githubusercontent.com/63149170/114365058-b12fd100-9b97-11eb-99a4-dc58cd797abc.png)

I was talking to me!
